flask_autodoc/templates/autodoc_default.html

```diff
@@ -57,9 +57,15 @@
         <div class="mapping">
             <a id="rule-{{doc.rule|urlencode}}" class="rule"><h2>{{doc.rule|escape}}</h2></a>
             <ul class="methods">
-                    {% for method in doc.methods -%}
-                    <li class="method">{{method}}</li>
-                    {% endfor %}
+                {% for method in doc.methods -%}
+                    {% if method == 'GET' and doc.args == ['None'] %}
+                        <a href="{{doc.rule}}" class="getmethod">
+                    {% endif %}
+                        <li class="method">{{method}}</li>
+                    {% if method == 'GET' %}
```

Comment:
Shouldn't you copy the same `if` condition on this line? What happened to `doc.args`?

It seems like an `if/else` would be easier to read, even if it means duplicating the `<li>` section.